docs/source/extensions/overview_extensions.rst

```diff
@@ -0,0 +1,44 @@
+Overview
+========
+
+ASReview has support for extensions, which are a nice way to extend the
+functionality of the :doc:`ASReview LAB <../lab/overview_lab>` software or the
+:doc:`command line interface <../API/cli>`. There are :ref:`officially
+supported extensions<extensions-official>` and :ref:`users'<extensions-users>`
+contributions. There is extensive documentation on how to add new :ref:`models
+<extensions-dev-model>` (i.e., classifiers, query strategies, balance
+strategies, or feature extraction techniques) or completely new
+:ref:`subcommands <extensions-dev-subcommand>`.
+
+
+.. _extensions-official:
+
+Offically Supported Extensions
+------------------------------
+
+
+The following extensions are officially supported and were developed as part
+of the core project:
+
+- :doc:`extension_covid19`: Make literature on COVID-19 available in ASReview.
+- :doc:`extension_visualization`: Plotting functionality for state files produced by ASReview.
+- :doc:`extension_wordcloud`: Create a visual impression of the contents of datasets via a wordcloud.
+- ``asreview-statistics``: Tool to give some basic properties of a dataset, such as number of
+  papers, number of inclusions. `GitHub <https://github.com/asreview/asreview-statistics>`__
+- ``asreview-hyperopt``: Optimize the hyperparameters of the models in ASReview. `GitHub <https://github.com/asreview/asreview-hyperopt>`__
+
+
+
+.. _extensions-users:
+
+Extensions Developed by Users
```

Comment:
```suggestion
Community-maintained extensions
```

I prefer to use "community-maintained", as it is more clearly a third party is maintaining it. 
See also https://help.ubuntu.com/community/Repositories/Ubuntu

Reply:
I agree, nice suggestion